akka-http-testkit/src/main/scala/akka/http/javadsl/testkit/WSProbe.scala

```diff
@@ -72,7 +72,9 @@ class WSProbe(delegate: st.WSProbe) {
 
   /**
    * Expect no message on the input side of the flow.
+   * @deprecated Use the variant with [[FiniteDuration]]
    */
+  @Deprecated
```

Comment:
I took the liberty to deprecate this variant without `FiniteDuration`. It was using `expectNoMsg` in Streams Testkit which is deprecated

Reply:
I don't understand this one? Now it's using `delegate.expectNoMessage()` which seems to be fine and fairly useful?

Reply:
It is because it relies on a deprecated method on the Streams Teskit:
https://github.com/akka/akka-http/blob/056b59824c36a3d0cdfca392ba57f9d407e64c2a/akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/WSProbe.scala#L133
`expectNoMessage` is in the end implemented with: `akka.stream.testkit.TestSubscriber.ManualProbe#expectNoMsg` which is deprecated. This is why I thought that it would make sense to deprecate this one as well.

Reply:
Thanks, I see. I'd say let's keep it still, it seems like an oversight that this particular overload is missing in streams testkit. It's still there in the actor testkit.

https://github.com/akka/akka/blob/e1a0a1fa3fec7650eb0f28027920357f773e3629/akka-testkit/src/main/scala/akka/testkit/TestKit.scala#L667

Maybe you can open a PR on akka/akka to add it back to the streams testkit as well?

Reply:
OK, then I'll remove the `@deprecated` annotation and open a PR on akka/akka

Reply:
Removed the `@deprecated` and created PR on Akka: https://github.com/akka/akka/pull/26070